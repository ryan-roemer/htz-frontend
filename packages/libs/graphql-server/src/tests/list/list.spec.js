import testServer from '../runTestServer';
import query from './listQuery.graphql';

it('fetches single list', async () => {
  // // run query against the server and snapshot the output
  const { errors, data, } = await testServer.query({ query, variables: { input: { listId: '7.7829323', }, }, });

  expect(errors).toBeUndefined();
  expect(data).toMatchSnapshot();
});
