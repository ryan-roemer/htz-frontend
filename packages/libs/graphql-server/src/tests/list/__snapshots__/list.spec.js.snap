// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetches single list 1`] = `
Object {
  "list": Object {
    "clickTrackers": Array [],
    "commercialLinks": null,
    "content": Array [],
    "contentId": "7.7829323",
    "contentName": "ליסט ספיישל",
    "description": "זהו תיאור האירוע המיוחד שבגללו יש ליסט ספיישל",
    "dfp": Array [],
    "extraLinks": null,
    "hasPagination": false,
    "inputTemplate": "com.tm.element.List",
    "isLazyloadImages": true,
    "items": Array [
      Object {
        "authorImage": null,
        "authors": null,
        "commentsCounts": 1,
        "contentId": "1.6652579",
        "exclusive": "Analysis ",
        "exclusiveMobile": null,
        "firstParagraph": null,
        "hash": null,
        "image": Object {
          "accessibility": "Hamas Chief Ismail Haniyeh shakes hands with Egyptian senior security official Ayman Badie during their meeting in Gaza City October 18, 2018. ",
          "aspects": null,
          "contentId": "1.6652637",
          "contentName": "Hamas Chief Ismail Haniyeh shakes hands with Egyptian senior security official Ayman Badie",
          "credit": "REUTERS/Mohammed Salem",
          "imageType": "image",
          "imageUrlLinkGroup": null,
          "imageUrlLinkGroupTarget": null,
          "imgArray": Array [
            Object {
              "aspects": Object {
                "full": Object {
                  "height": 1405,
                  "width": 2200,
                },
                "headline": Object {
                  "height": 1280,
                  "width": 2200,
                  "x": 0,
                  "y": 125,
                },
                "regular": Object {
                  "height": 1405,
                  "width": 1873,
                  "x": 281,
                  "y": 0,
                },
                "square": Object {
                  "height": 1405,
                  "width": 1406,
                  "x": 391,
                  "y": 0,
                },
              },
              "imgName": "image/3337836633.jpg",
              "version": "1542115336",
            },
          ],
          "inputTemplate": "com.tm.Image",
          "isAnimated": null,
          "kind": "image",
          "photographer": "none",
          "position": null,
          "title": "Hamas Chief Ismail Haniyeh shakes hands with Egyptian senior security official Ayman Badie during their meeting in Gaza City October 18, 2018. ",
          "viewMode": null,
        },
        "inputTemplate": "com.tm.TeaserData",
        "isPremiumContent": null,
        "lastUpdate": null,
        "media": null,
        "mediaFlags": null,
        "path": "http://pre.haaretz.com/israel-news/hamas-tries-to-dictate-new-rules-of-game-in-gaza-but-it-may-be-misjudging-israel-1.6652579",
        "publishDate": null,
        "rank": null,
        "relatedArticles": null,
        "representedContent": "1.6652579",
        "representedContentType": "regularArticle",
        "subtitle": null,
        "subtitleMobile": null,
        "title": "RRRR Israel's early elections: How Trump accidentally exposed Netanyahu's bluff",
        "titleMobile": "Hamas tries to dictate new rules of game in Gaza, but it may be misjudging Israel",
      },
      Object {
        "authorImage": null,
        "authors": null,
        "commentsCounts": 0,
        "contentId": "1.6650718",
        "exclusive": null,
        "exclusiveMobile": null,
        "firstParagraph": null,
        "hash": null,
        "image": null,
        "inputTemplate": "com.tm.TeaserData",
        "isPremiumContent": null,
        "lastUpdate": null,
        "media": null,
        "mediaFlags": null,
        "path": "http://pre.haaretz.co.il/captain/viral/1.6650718",
        "publishDate": null,
        "rank": null,
        "relatedArticles": null,
        "representedContent": "1.6650718",
        "representedContentType": "regularArticle",
        "subtitle": null,
        "subtitleMobile": null,
        "title": "כשהפארודיה ירתה לעצמה בראש: הכירו את ה\\"טראמפי בר\\", הדובי הנשיאותי",
        "titleMobile": "כשהפארודיה ירתה לעצמה בראש: הכירו את ה\\"טראמפי בר\\", הדובי הנשיאותי",
      },
      Object {
        "authorImage": null,
        "authors": null,
        "commentsCounts": 48,
        "contentId": "1.5746469",
        "exclusive": null,
        "exclusiveMobile": null,
        "firstParagraph": null,
        "hash": null,
        "image": null,
        "inputTemplate": "com.tm.TeaserData",
        "isPremiumContent": null,
        "lastUpdate": null,
        "media": null,
        "mediaFlags": null,
        "path": "http://pre.haaretz.co.il/news/local/EXT-MAGAZINE-1.5746469",
        "publishDate": null,
        "rank": null,
        "relatedArticles": null,
        "representedContent": "1.5746469",
        "representedContentType": "magazineArticle",
        "subtitle": null,
        "subtitleMobile": null,
        "title": "ירושלים של מים: הצצה לפרויקט הענק להקמת מנהרת המים הראשונה בישראל",
        "titleMobile": "ירושלים של מים: הצצה לפרויקט הענק להקמת מנהרת המים הראשונה בישראל",
      },
      Object {
        "authorImage": null,
        "authors": null,
        "commentsCounts": 1,
        "contentId": "1.6652134",
        "exclusive": null,
        "exclusiveMobile": null,
        "firstParagraph": null,
        "hash": null,
        "image": Object {
          "accessibility": "\\"ערב טוב עם גיא פינס\\". פינס ונשות התוכנית",
          "aspects": null,
          "contentId": "1.6652133",
          "contentName": "ערב טוב עם גיא פינס",
          "credit": "גיא כושי ויריב פיין�",
          "imageType": "image",
          "imageUrlLinkGroup": null,
          "imageUrlLinkGroupTarget": null,
          "imgArray": Array [
            Object {
              "aspects": Object {
                "belgrade": Object {
                  "height": 631,
                  "width": 2010,
                  "x": 0,
                  "y": 242,
                },
                "full": Object {
                  "height": 1546,
                  "width": 2048,
                },
                "headline": Object {
                  "height": 1191,
                  "width": 2048,
                  "x": 0,
                  "y": 301,
                },
                "landscape": Object {
                  "height": 861,
                  "width": 1994,
                  "x": 0,
                  "y": 313,
                },
                "regular": Object {
                  "height": 1366,
                  "width": 1822,
                  "x": 0,
                  "y": 96,
                },
                "square": Object {
                  "height": 1546,
                  "width": 1546,
                  "x": 276,
                  "y": 0,
                },
                "vertical": Object {
                  "height": 1487,
                  "width": 1266,
                  "x": 222,
                  "y": 13,
                },
              },
              "imgName": "image/94157394.jpg",
              "version": "1542102616",
            },
          ],
          "inputTemplate": "com.tm.Image",
          "isAnimated": null,
          "kind": "image",
          "photographer": "none",
          "position": null,
          "title": "\\"ערב טוב עם גיא פינס\\". בלי טיפת אינפוט מערכתי",
          "viewMode": null,
        },
        "inputTemplate": "com.tm.TeaserData",
        "isPremiumContent": null,
        "lastUpdate": null,
        "media": null,
        "mediaFlags": null,
        "path": "http://pre.haaretz.co.il/gallery/television/lazy-eye/1.6652134",
        "publishDate": null,
        "rank": null,
        "relatedArticles": null,
        "representedContent": "1.6652134",
        "representedContentType": "regularArticle",
        "subtitle": null,
        "subtitleMobile": null,
        "title": "המעבר לקשת חיסל את הפוזה של גיא פינס",
        "titleMobile": "המעבר לקשת חיסל את הפוזה של גיא פינס",
      },
      Object {
        "authorImage": null,
        "authors": null,
        "commentsCounts": 13,
        "contentId": "1.5762713",
        "exclusive": null,
        "exclusiveMobile": null,
        "firstParagraph": null,
        "hash": null,
        "image": Object {
          "accessibility": "ריטמן בדיון בעניינו בבג\\"ץ, ביולי",
          "aspects": null,
          "contentId": "1.5762723",
          "contentName": "ריטמן בדיון בעניינו בבג\\"ץ, ביולי",
          "credit": null,
          "imageType": "image",
          "imageUrlLinkGroup": null,
          "imageUrlLinkGroupTarget": null,
          "imgArray": Array [
            Object {
              "aspects": Object {
                "belgrade": Object {
                  "height": 679,
                  "width": 2164,
                  "x": 36,
                  "y": 226,
                },
                "full": Object {
                  "height": 1468,
                  "width": 2200,
                },
                "headline": Object {
                  "height": 1254,
                  "width": 2156,
                  "x": 44,
                  "y": 214,
                },
                "landscape": Object {
                  "height": 893,
                  "width": 2069,
                  "x": 131,
                  "y": 226,
                },
                "regular": Object {
                  "height": 758,
                  "width": 1009,
                  "x": 643,
                  "y": 202,
                },
                "square": Object {
                  "height": 1389,
                  "width": 1390,
                  "x": 520,
                  "y": 79,
                },
              },
              "imgName": "image/95458065.jpg",
              "version": "1516728833",
            },
          ],
          "inputTemplate": "com.tm.Image",
          "isAnimated": null,
          "kind": "image",
          "photographer": "none",
          "position": null,
          "title": "ריטמן בדיון בעניינו בבג\\"ץ, ביולי",
          "viewMode": null,
        },
        "inputTemplate": "com.tm.TeaserData",
        "isPremiumContent": null,
        "lastUpdate": null,
        "media": null,
        "mediaFlags": null,
        "path": "http://pre.haaretz.co.il/news/law/.premium-MAGAZINE-1.5762713",
        "publishDate": null,
        "rank": null,
        "relatedArticles": null,
        "representedContent": "1.5762713",
        "representedContentType": "magazineArticle",
        "subtitle": null,
        "subtitleMobile": null,
        "title": "ניצב רוני ריטמן, שנחשד בהטרדה, ביקש מהמפכ\\"ל לעבור מתפקיד מפקד להב 433",
        "titleMobile": "ניצב רוני ריטמן, שנחשד בהטרדה, ביקש מהמפכ\\"ל לעבור מתפקיד מפקד להב 433",
      },
    ],
    "loadPriority": "client",
    "marketingTeaser": null,
    "title": "ספיישל",
    "url": null,
    "urlDescription": null,
    "view": "Mom",
  },
}
`;
